src/Config.ts

```diff
@@ -0,0 +1,135 @@
+import * as fs from 'fs-extra';
+import { Account } from './Account';
+import { MosaicConfig } from './MosaicConfig';
+
+// Database password key to read from env.
+const DB_PASSWORD = 'MOSAIC_FACILITATOR_DB_PASSWORD';
+
+// Database type
+enum DBType {
+  SQLITE = 'SQLITE',
+}
+
+
+/**
+ * Holds mosaic config, database config and facilitator config.
+ */
+export class Config {
+  public database: DBConfig;
+
+  public chains: Record<string, any>;
+
+  public mosaic: MosaicConfig;
+
+  /**
+   * Constructor.
+   * @param mosaicConfigPath Mosaic config path.
+   * @param facilitatorConfigPath Facilitator config path.
+   */
+  public constructor(
+    mosaicConfigPath: string,
+    facilitatorConfigPath: string,
+  ) {
+    this.mosaic = MosaicConfig.fromPath(mosaicConfigPath);
+
+    const facilitatorConfig = this.getConfigJsonFromPath(
+      facilitatorConfigPath,
+    );
+
+    this.database = new DBConfig(facilitatorConfig);
+    this.chains = this.getChains(facilitatorConfig);
+  }
+
+  /**
+   * Get chain objects from given facilitator config.
+   * @param facilitatorConfig Facilitator config json data.
+   * @returns Array containing chain object.
+   */
+  private getChains(facilitatorConfig: Record<string, any>): Chain[] {
+    const chainsJson: Record<string, any> = facilitatorConfig.chains;
+    const chains: Chain[] = [];
+    for (const key in chainsJson) {
+      chains.push(new Chain(key, facilitatorConfig));
+    }
+    return chains;
+  }
+
+  /**
+   * Get config json data from the given file path.
+   * @param filePath Facilitator config file path.
+   * @returns JSON data for facilitator config.
+   */
+  private getConfigJsonFromPath(filePath: string): Record<string, any> {
```

Comment:
`getManifestFromPath` and should be in yaml?

Reply:
User may not add the accounts and encrypted keys manually. They are generated by the code.
The encrypted account will be JSON, so it will become a mix of YAML and JSON as a string.

I still think that its config.
The file defined the configuration to run the facilitator.

Reply:
Ok, makes sense